ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/builder/FuluBuilder.java

```diff
@@ -107,13 +107,30 @@ public FuluBuilder kzgCommitmentsInclusionProofDepth(
   }
 
   public FuluBuilder blobSchedule(final List<BlobScheduleEntry> blobSchedule) {
-    checkNotNull(this.blobSchedule);
+    checkNotNull(blobSchedule);
+    verifyBlobSchedule(blobSchedule);
     this.blobSchedule.clear();
     // copy list rather than use the one passed in case we need to add to the list during validation
     this.blobSchedule.addAll(blobSchedule);
     return this;
   }
 
+  public void verifyBlobSchedule(final List<BlobScheduleEntry> blobSchedule) {
+    if (blobSchedule.isEmpty()) {
+      return;
+    }
+    BlobScheduleEntry previousEntry = blobSchedule.getFirst();
+    for (final BlobScheduleEntry entry : blobSchedule.subList(1, blobSchedule.size())) {
+      if (!entry.epoch().isGreaterThan(previousEntry.epoch())) {
+        throw new IllegalArgumentException(
+            String.format(
+                "Blob schedule must be ordered and doesn't contain duplicates, while %s and %s entries were provided.",
```

Comment:
we were previously just sorting on epoch, im not sure where that went but imo sorting is probably better. we just want to error if theres the same epoch re-specified.

we want to also be careful here with user facing wording, if we want to do this particular check, we need more like`Blob schedule must be in ascending order, and not contain duplicates.  %s and %s in the provided blob schedule do not follow this requirement.`